clarifai.go

```diff
@@ -0,0 +1,245 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"sort"
+	"strings"
+)
+
+const (
+	clarifaiApi = "https://api.clarifai.com/v2/models/aaa03c23b3724a16a56b629203edc62c/outputs"
+	clarifyAuth = "https://api.clarifai.com/v1/token"
+	minValue    = 0.72
+	maxTags     = 10
+)
+
+type ClarifaiClient struct {
+	Token  string
+	Expiry int
+	Access bool
+}
+
+type ClarifyTag struct {
+	Name  string  `json:"name"`
+	Value float64 `json:"value"`
+}
+
+func NewClarifaiClient() *ClarifaiClient {
+	return &ClarifaiClient{
+		Access: false,
+	}
+}
+
+// Check whether the client still has access rights to Clarifai
+func (client *ClarifaiClient) isAccess() bool {
+	if !client.Access || client.Expiry <= 0 {
```

Comment:
it's not, as i said this is still a work-in-progress. I was already tired of all the refactoring so I just put it there to get the idea across :). And yeah we can do that, but ideally i would like to remove all that from the request and have a function that does that automatically